### DISCUSSION TITLE
v1.10.0

### DISCUSSION THREAD

**Discussion by cdubz** [babybuddy/babybuddy#392] [General]
| :exclamation: Configuration change :exclamation: |
|:---------------------------|
| If you are using Baby Buddy behind a proxy this version may cause a CSRF error 403 when attempting to add new entries. Set the `SECURE_PROXY_SSL_HEADER` environment variable to `True` (or any non-falsy value) in your deployment to resolve this. This is correct behavior as of Django v4.0 (used by Baby Buddy v1.10.0+). See #393 for more information.  |

[Full Changelog](https://github.com/babybuddy/babybuddy/compare/v1.10.0...HEAD)

**Implemented enhancements:**

- Hardcoded English strings on timeline [\#352](https://github.com/babybuddy/babybuddy/issues/352)
- Feature Request:  Track baby height [\#191](https://github.com/babybuddy/babybuddy/issues/191)
- breakout feeding types [\#384](https://github.com/babybuddy/babybuddy/pull/384) ([alzyee](https://github.com/alzyee))
- Rearrange dashboard cards to set timer card first [\#382](https://github.com/babybuddy/babybuddy/pull/382) ([adamaze](https://github.com/adamaze))
- Update to Django 4.x [\#378](https://github.com/babybuddy/babybuddy/pull/378) ([cdubz](https://github.com/cdubz))
- Updated Italian translation [\#376](https://github.com/babybuddy/babybuddy/pull/376) ([nos86](https://github.com/nos86))
- Fix ordering of start/stop items when start and stop times are the same [\#372](https://github.com/babybuddy/babybuddy/pull/372) ([MrApplejuice](https://github.com/MrApplejuice))
- Fix German translation mistake [\#368](https://github.com/babybuddy/babybuddy/pull/368) ([MrApplejuice](https://github.com/MrApplejuice))
- Add timer restart and stop triggers to REST-API [\#367](https://github.com/babybuddy/babybuddy/pull/367) ([MrApplejuice](https://github.com/MrApplejuice))
- Optional last name [\#361](https://github.com/babybuddy/babybuddy/pull/361) ([Alberdi](https://github.com/Alberdi))
- Add Height, Head Circumference, and BMI [\#360](https://github.com/babybuddy/babybuddy/pull/360) ([Daegalus](https://github.com/Daegalus))
- Improve iOS webapp/clip/pwa experience [\#359](https://github.com/babybuddy/babybuddy/pull/359) ([cdubz](https://github.com/cdubz))
- Convert envir variables that are supposed to be boolean to boolean [\#356](https://github.com/babybuddy/babybuddy/pull/356) ([MagiX13](https://github.com/MagiX13))

**Fixed bugs:**

- iOS 15 web app experience is degraded [\#357](https://github.com/babybuddy/babybuddy/issues/357)
- Boolean environmental variables  [\#354](https://github.com/babybuddy/babybuddy/issues/354)
- Sleep graph has incorrect positioning when there is a gap in days. [\#286](https://github.com/babybuddy/babybuddy/issues/286)
- Sleep graph issues [\#283](https://github.com/babybuddy/babybuddy/issues/283)
- KeyError at /children/XXX/reports/sleep/pattern/ [\#211](https://github.com/babybuddy/babybuddy/issues/211)
- fix\(sleep-reports\): \#286 Init all days in the period to remove gaps [\#341](https://github.com/babybuddy/babybuddy/pull/341) ([codisart](https://github.com/codisart))

**Closed issues:**

- Show type breakdown on feeding\_amounts report [\#383](https://github.com/babybuddy/babybuddy/issues/383)
- Error during add/edit action for fed and diaper [\#374](https://github.com/babybuddy/babybuddy/issues/374)
- Evaluate replacements for Easy Thumbnails [\#373](https://github.com/babybuddy/babybuddy/issues/373)
- Issue with timer API for Feeding, Sleep, and Tummy-Time [\#363](https://github.com/babybuddy/babybuddy/issues/363)

**Merged pull requests:**

- Hide feeding\_day card when no information present [\#386](https://github.com/babybuddy/babybuddy/pull/386) ([BenjaminHae](https://github.com/BenjaminHae))
- add HA Addon link [\#375](https://github.com/babybuddy/babybuddy/pull/375) ([OttPeterR](https://github.com/OttPeterR))
- Minor changes to run into subdir [\#358](https://github.com/babybuddy/babybuddy/pull/358) ([MagiX13](https://github.com/MagiX13))

<hr /><em>This discussion was created from the release <a href='https://github.com/babybuddy/babybuddy/releases/tag/v1.10.0'>v1.10.0</a>.</em>

**Comment by Palleri** [comment]
When I try to add a child I get the verfication error again. I managed to get in to the app after I added your configuration.
